pyproject.toml

```diff
@@ -0,0 +1,4 @@
+[tool.towncrier]
+    package = "aioredis"
+    package_dir = ""
+    filename = "NEWS.rst"
```

Comment:
Lets use existing changelog `CHANGES.rst`
Also put the `.. towncrier release notes start` below `Changes` headline